chemprop/data/utils.py

```diff
@@ -859,7 +819,22 @@ def split_data(data: MoleculeDataset,
 
         return MoleculeDataset(train), MoleculeDataset(val), MoleculeDataset(test)
     elif split_type == 'molecular_weight':
-        train, val, test = weight_split(data, bias='small', sizes=sizes)
+        train_size, val_size, test_size = sizes[0] * len(data), sizes[1] * len(data), sizes[2] * len(data)
```

Comment:
```suggestion
        train_size, val_size, _ = [int(size * len(data)) for size in sizes]
```
I don't think `test_size` is actually used?

Reply:
I think we can leave it up to the user defined split sizes. I could leave the val and test unsorted, so the val distribution is not far from the test.